relations/templates/relations/suppress.html

```diff
@@ -0,0 +1,5 @@
+{%- extends "relations/base.html" %}
+
+{% block content %}
+A relation {{ relation_id }} has been suppressed by a new relation for id {{ arxiv_id }} version {{ arxiv_ver }} with description {{ description }}
```

Comment:
Ultimately we may not need to show the user so much about how the sausage is made (i.e. from their standpoint the operation would look like deletion). But this is fine for now, and we can build on it.